src/nginx/nginx.yaml

```diff
@@ -0,0 +1,239 @@
+    http:
+      paths:
+      - path: /.well-known/simple-web-discovery
+        backend:
+          serviceName: oxauth
+          servicePort: 8080
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: gluu-ingress-scim-configuration
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "300"
+    nginx.ingress.kubernetes.io/configuration-snippet: "rewrite /.well-known/scim-configuration /identity/restv1/scim-configuration$1 break;"
+    nginx.ingress.kubernetes.io/rewrite-target: /identity/restv1/scim-configuration
+spec:
+  tls:
+    - hosts:
+      - eoepca-dev.gluu.org
+      secretName: tls-certificate
+  rules:
+  - host: eoepca-dev.gluu.org
+    http:
+      paths:
+      - path: /.well-known/scim-configuration
+        backend:
+          serviceName: oxtrust
+          servicePort: 8080
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: gluu-ingress-fido-u2f-configuration
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "300"
+    nginx.ingress.kubernetes.io/configuration-snippet: "rewrite /.well-known/fido-u2f-configuration /oxauth/restv1/fido-u2f-configuration$1 break;"
+    nginx.ingress.kubernetes.io/rewrite-target: /oxauth/restv1/fido-u2f-configuration
+spec:
+  tls:
+    - hosts:
+      - eoepca-dev.gluu.org
+      secretName: tls-certificate
+  rules:
+  - host: eoepca-dev.gluu.org
+    http:
+      paths:
+      - path: /.well-known/fido-u2f-configuration
+        backend:
+          serviceName: oxauth
+          servicePort: 8080
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: gluu-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.org/ssl-services: "oxauth"
+    nginx.ingress.kubernetes.io/proxy-next-upstream: "error timeout invalid_header http_500 http_502 http_503 http_504"
+spec:
+  tls:
+    - hosts:
+      - eoepca-dev.gluu.org
+      secretName: tls-certificate
+  rules:
+  - host: eoepca-dev.gluu.org
+    http:
+      paths:
+      - path: /oxauth
+        backend:
+          serviceName: oxauth
+          servicePort: 8080
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: gluu-ingress-stateful
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.org/ssl-services: "oxtrust"
+    nginx.ingress.kubernetes.io/app-root: "/identity"
+    nginx.ingress.kubernetes.io/affinity: "cookie"
+    nginx.ingress.kubernetes.io/session-cookie-name: "route"
+    nginx.ingress.kubernetes.io/session-cookie-hash: "sha1"
+    nginx.ingress.kubernetes.io/proxy-next-upstream: "error timeout invalid_header http_500 http_502 http_503 http_504"
+spec:
+  tls:
+    - hosts:
+      - eoepca-dev.gluu.org
+      secretName: tls-certificate
+  rules:
+  - host: eoepca-dev.gluu.org
```

Comment:
If at all possible, we should make any host/dns configurable, as this is the same code that will be executed in production, or even other platforms, and we can't be sure of the hostname

Reply:
Changed this to a placeholder variable, that is replaced at runtime in setupCluster.sh. Variable is defined in the script.